### DISCUSSION TITLE
[BUG] .flake8 file is ignored

### DISCUSSION THREAD

**Discussion by niloatgoshare** [s-weigand/flake8-nb#242] [Q&A] [answered]
**Describe the bug**
The configuration set inside the .flake8 file is being ignored.

**To Reproduce**
Create test.py
```
def this_is_a_super_long_method_name_for_a_function_this_is_a_super_long_method():
    pass

```

.flake8
```
[flake8]
ignore =
    E501
max-line-length = 88
```

**Expected behavior**
The error E501 should not be raised. 

**Desktop (please complete the following information):**

- OS: ubuntu
- Version 22.04 LTS

**Additional context**
Running flake8 works as intended.
Running `flake8-nb --config .flake8` or `flake8-nb --append-config .flake8` wont not work either.

**Comment by s-weigand** [comment, marked as answer]
Since `flake8-nb` highjacks the `flake8` internals (in this case application name) you need to use `[flake8_nb]` instead of `[flake8]` in your config file. 
If you also name the config file `.flake8_nb` instead of `.flake8` it should be autodetected. 
See [usage docs](https://flake8-nb.readthedocs.io/en/latest/usage.html#project-wide-configuration)
Feel free to close the issue if that solves your problem

**Comment by niloatgoshare** [comment]
Thanks, appreciate it
